package.json

```diff
@@ -52,15 +52,16 @@
   },
   "prettier": {},
   "dependencies": {
-    "retext": "^8.1.0",
-    "retext-smartypants": "^5.2.0",
+    "retext": "^9.0.0",
+    "retext-smartypants": "^6.0.0",
     "unist-util-visit": "^5.0.0"
   },
   "devDependencies": {
+    "remark-mdx": "^3.0.0",
     "nano-staged": "^0.8.0",
     "prettier": "^3.1.1",
-    "remark": "^14.0.3",
+    "remark": "^15.0.1",
     "simple-git-hooks": "^2.9.0",
-    "vitest": "^1.1.2"
+    "vitest": "^1.1.3"
```

Comment:
These dependencies version up  (including L55, L56) are needed for this feature? 
Otherwise, I think it would be nicer to be separated to other PR.

Reply:
I did a rollback for `L55, L56`